content/components/table/default.md

```diff
@@ -1,3 +1,58 @@
-We're currently designing a basic responsive HTML table component. Join the
-conversation in our
-[community forum](https://community.digital.gov.au/t/table).
+---
+layout: component/code-demo
+iframe: examples/example-default
+iframeFullwidth: true
+code:
+  - HTML: |
+        <table class="au-table">
+            <caption class="au-table__caption">Population of Australian states and territories, December 2015</caption>
+            <thead class="au-table__head">
```

Comment:
Anticipating questions on <col> and <colgroup> - do we want to explain why these weren't in scope of the initial release?